package.json

```diff
@@ -54,16 +54,16 @@
     "vuex-class-modules": "^1.1.2"
   },
   "devDependencies": {
-    "@babel/core": "7.8.7",
+    "@babel/core": "7.9.0",
```

Comment:
Why you are updating everything in cases, when we need only one package update?

Reply:
just up to date all deps and for no any notify in repo about security error with `old dependency` or something else

Reply:
Security errors better solve through special PR from dependabot. When you are updating dependencies directly it not exclude similar cases with security issues

Reply:
Then, just update ones what you need, not all packages

Reply:
We should use latest versions, soon as possible